src/main/kotlin/me/bristermitten/devdenbot/extensions/Messages.kt

```diff
@@ -12,6 +15,29 @@ fun CommandEvent.arguments(): Arguments {
         , args.filter { !it.isFlag }, args.filter { it.isFlag })
 }
 
+suspend fun Message.getMentionedMember(index: Int, action: suspend (Member?) -> Unit = {}) {
+    val str = contentRaw.split(WHITESPACE_REGEX).getOrNull(index) ?: let {
```

Comment:
Probably want to use run {} here, rather than let {}

Reply:
i just did `?: return null`